theano/tensor/nnet/__init__.py

```diff
@@ -150,3 +151,120 @@ def conv2d(input, filters, input_shape=None, filter_shape=None,
     return abstract_conv2d(input, filters, input_shape, filter_shape,
                            border_mode, subsample, filter_flip,
                            filter_dilation)
+
+
+def conv3d(input, filters, input_shape=None, filter_shape=None,
+           border_mode='valid', subsample=(1, 1, 1), filter_flip=True,
+           image_shape=None, filter_dilation=(1, 1, 1), **kwargs):
```

Comment:
Since you don't need to provide backwards compatibility because this is a new function you can just import conv3d and not have a wrapper function.  The checks in conv2d are only there because there was another conv2d function that got replaced.

You should probably move the documentation though while dropping references to deprecated things.